releasenotes/notes/removed_ddtrace.util.py-0d0f48aefa6d9779.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed ``ddtrace.util`` module. Please use ``ddtrace.utils`` instead.
```

Comment:
We're getting rid of ddtrace.utils as well. We should just say it's being removed. No one should be using anything from here anymore anyway